frontend/src/pages/Buttons/Buttons.tsx

```diff
@@ -129,6 +130,48 @@ export const Buttons = () => {
 					This is ignored.{' '}
 					<img src={Logo} height={16} alt="ignored" />
 				</div>
+				<button
+					onClick={() => {
+						for (const id of ['1', '2', '3']) {
+							const elem = document.getElementById(`email-${id}`)!
+							elem.innerText = `${(Math.random() + 1)
+								.toString(36)
+								.substring(7)} test@test.com ${(
+								Math.random() + 1
+							)
+								.toString(36)
+								.substring(7)}`
+						}
+						setEmail(
+							`${(Math.random() + 1)
+								.toString(36)
+								.substring(7)} test@test.com ${(
+								Math.random() + 1
+							)
+								.toString(36)
+								.substring(7)}`,
+						)
+					}}
+				>
+					set email
+				</button>
+				<div>
+					<b>hello, this is a b tag</b>
+					<b key="email-1">{email}</b>
```

Comment:
@Vadman97 have you been able to reproduce the bug? I've been messing with some of the state, but the elements are being obfuscated through the `adds` nodes rather than the `texts`. Been messing with some css to see if I can reproduce better

Reply:
got it to repro not by using the react state, but by manually setting `innerText` see https://github.com/highlight/highlight/blob/669886629b7bfdefa079c5c097899a2e7ceabb6e/frontend/src/pages/Buttons/Buttons.tsx#L137-L141 and the screenshots above

Reply:
i'm guessing the customer's app is using jquery or some other library to directly edit the DOM innerText, hence we are seeing this bug for them